### ISSUE TITLE
Restore of restic volume fails with "No ID found"

### ISSUE THREAD

**Issue by tarioch** [heptio/ark#896]
**What steps did you take and what happened:**
I have a pod with a pvc, backup seems to run fine but when I try to restore the backup it fails with

```
time="2018-10-02T20:48:40Z" level=error msg="Error restoring volume" controller=pod-volume-restore error="error running restic restore, cmd=restic restore --repo=s3:http://minio:9000/tarioark-restic/test --password-file=/tmp/ark-restic-credentials-test795185276 --cache-dir=/scratch/.cache/restic 3548b30b --target=., stdout=, stderr=invalid id \"3548b30b\": no ID found\n: exit status 1" error.file="pod_volume_restore_controller.go:320" error.function=restorePodVolume logSource="pkg/controller/pod_volume_restore_controller.go:287" name=full-20181002030055-20181002223210-k2mqj namespace=heptio-ark restore=heptio-ark/full-20181002030055-20181002223210
```

It looks like there is data stored in the restic bucket. I'm playing around with restoring and doing a remapping of the namespace (basically want to test restore without impacting the original pod)

**What did you expect to happen:**

The pod should be restored.

**The output of the following commands will help us better understand what's going on**:
(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)

* `kubectl logs deployment/ark -n heptio-ark`
```
time="2018-10-02T20:48:40Z" level=info msg="restore completed" backup=full-20181002030055 logSource="pkg/controller/restore_controller.go:432" restore=heptio-ark/full-20181002030055-20181002223210
```
* `ark backup describe <backupname>` or `kubectl get backup/<backupname> -n heptio-ark -o yaml`
```
Name:         full-20181002030055
Namespace:    heptio-ark
Labels:       ark-schedule=full
Annotations:  <none>

Phase:  Completed

Namespaces:
  Included:  *
  Excluded:  <none>

Resources:
  Included:        *
  Excluded:        <none>
  Cluster-scoped:  auto

Label selector:  <none>

Snapshot PVs:  auto

TTL:  720h0m0s

Hooks:  <none>

Backup Format Version:  1

Started:    2018-10-02 05:00:55 +0200 CEST
Completed:  2018-10-02 05:03:43 +0200 CEST

Expiration:  2018-11-01 04:00:55 +0100 CET

Validation errors:  <none>

Persistent Volumes: <none included>

Restic Backups:
  Completed:
    default/mysqlbackup-6b7bbbcb84-txbg7: backupstorage
    default/prometheus-kube-prometheus-0: prometheus-kube-prometheus-db
```
* `ark restore describe <restorename>` or `kubectl get restore/<restorename> -n heptio-ark -o yaml`
```
Name:         full-20181002030055-20181002223210
Namespace:    heptio-ark
Labels:       <none>
Annotations:  <none>

Backup:  full-20181002030055

Namespaces:
  Included:  *
  Excluded:  <none>

Resources:
  Included:        *
  Excluded:        nodes, events, events.events.k8s.io, backups.ark.heptio.com, restores.ark.heptio.com
  Cluster-scoped:  auto

Namespace mappings:  default=test

Label selector:  app=mysqlbackup

Restore PVs:  auto

Phase:  Completed

Validation errors:  <none>

Warnings:  <none>
Errors:    <none>

Restic Restores:
  Failed:
    test/mysqlbackup-6b7bbbcb84-txbg7: backupstorage
```
* `ark restore logs <restorename>`
```
time="2018-10-02T20:32:14Z" level=info msg="Not including resource" groupResource=nodes logSource="pkg/restore/restore.go:124"
time="2018-10-02T20:32:14Z" level=info msg="Not including resource" groupResource=events logSource="pkg/restore/restore.go:124"
time="2018-10-02T20:32:14Z" level=info msg="Not including resource" groupResource=events.events.k8s.io logSource="pkg/restore/restore.go:124"
time="2018-10-02T20:32:14Z" level=info msg="Not including resource" groupResource=restores.ark.heptio.com logSource="pkg/restore/restore.go:124"
time="2018-10-02T20:32:14Z" level=info msg="Not including resource" groupResource=backups.ark.heptio.com logSource="pkg/restore/restore.go:124"
time="2018-10-02T20:32:15Z" level=info msg="Starting restore of backup heptio-ark/full-20181002030055" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring cluster level resource 'persistentvolumes' from: /tmp/536302380/resources/persistentvolumes/cluster" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'persistentvolumeclaims' into namespace 'test' from: /tmp/536302380/resources/persistentvolumeclaims/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'secrets' into namespace 'test' from: /tmp/536302380/resources/secrets/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'secrets' into namespace 'heptio-ark' from: /tmp/536302380/resources/secrets/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'secrets' into namespace 'kube-public' from: /tmp/536302380/resources/secrets/namespaces/kube-public" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'secrets' into namespace 'kube-system' from: /tmp/536302380/resources/secrets/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'secrets' into namespace 'rook-ceph' from: /tmp/536302380/resources/secrets/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'secrets' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/secrets/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'configmaps' into namespace 'test' from: /tmp/536302380/resources/configmaps/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'configmaps' into namespace 'heptio-ark' from: /tmp/536302380/resources/configmaps/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'configmaps' into namespace 'kube-public' from: /tmp/536302380/resources/configmaps/namespaces/kube-public" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'configmaps' into namespace 'kube-system' from: /tmp/536302380/resources/configmaps/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'configmaps' into namespace 'rook-ceph' from: /tmp/536302380/resources/configmaps/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'configmaps' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/configmaps/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'serviceaccounts' into namespace 'test' from: /tmp/536302380/resources/serviceaccounts/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'serviceaccounts' into namespace 'heptio-ark' from: /tmp/536302380/resources/serviceaccounts/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'serviceaccounts' into namespace 'kube-public' from: /tmp/536302380/resources/serviceaccounts/namespaces/kube-public" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'serviceaccounts' into namespace 'kube-system' from: /tmp/536302380/resources/serviceaccounts/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'serviceaccounts' into namespace 'rook-ceph' from: /tmp/536302380/resources/serviceaccounts/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'serviceaccounts' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/serviceaccounts/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'pods' into namespace 'test' from: /tmp/536302380/resources/pods/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Getting client for /v1, Kind=Pod" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Executing item action for pods" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Executing resticRestoreAction" kind=restoreitemaction logSource="pkg/restore/restic_restore_action.go:64" name=restic pluginName=ark
time="2018-10-02T20:32:15Z" level=info msg="Restic snapshot ID annotations found" kind=restoreitemaction logSource="pkg/restore/restic_restore_action.go:80" name=restic pluginName=ark pod=default/mysqlbackup-6b7bbbcb84-txbg7
time="2018-10-02T20:32:15Z" level=info msg="Done executing resticRestoreAction" kind=restoreitemaction logSource="pkg/restore/restic_restore_action.go:125" name=restic pluginName=ark
time="2018-10-02T20:32:15Z" level=info msg="Executing item action for pods" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring Pod: mysqlbackup-6b7bbbcb84-txbg7" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'pods' into namespace 'heptio-ark' from: /tmp/536302380/resources/pods/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'pods' into namespace 'kube-system' from: /tmp/536302380/resources/pods/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'pods' into namespace 'rook-ceph' from: /tmp/536302380/resources/pods/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'pods' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/pods/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'alertmanagers.monitoring.coreos.com' into namespace 'test' from: /tmp/536302380/resources/alertmanagers.monitoring.coreos.com/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring cluster level resource 'apiservices.apiregistration.k8s.io' from: /tmp/536302380/resources/apiservices.apiregistration.k8s.io/cluster" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'certificates.certmanager.k8s.io' into namespace 'test' from: /tmp/536302380/resources/certificates.certmanager.k8s.io/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring cluster level resource 'clusterissuers.certmanager.k8s.io' from: /tmp/536302380/resources/clusterissuers.certmanager.k8s.io/cluster" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring cluster level resource 'clusterrolebindings.rbac.authorization.k8s.io' from: /tmp/536302380/resources/clusterrolebindings.rbac.authorization.k8s.io/cluster" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring cluster level resource 'clusterroles.rbac.authorization.k8s.io' from: /tmp/536302380/resources/clusterroles.rbac.authorization.k8s.io/cluster" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'clusters.ceph.rook.io' into namespace 'rook-ceph' from: /tmp/536302380/resources/clusters.ceph.rook.io/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'configs.ark.heptio.com' into namespace 'heptio-ark' from: /tmp/536302380/resources/configs.ark.heptio.com/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'controllerrevisions.apps' into namespace 'test' from: /tmp/536302380/resources/controllerrevisions.apps/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'controllerrevisions.apps' into namespace 'heptio-ark' from: /tmp/536302380/resources/controllerrevisions.apps/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'controllerrevisions.apps' into namespace 'kube-system' from: /tmp/536302380/resources/controllerrevisions.apps/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'controllerrevisions.apps' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/controllerrevisions.apps/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring cluster level resource 'customresourcedefinitions.apiextensions.k8s.io' from: /tmp/536302380/resources/customresourcedefinitions.apiextensions.k8s.io/cluster" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'daemonsets.apps' into namespace 'test' from: /tmp/536302380/resources/daemonsets.apps/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'daemonsets.apps' into namespace 'heptio-ark' from: /tmp/536302380/resources/daemonsets.apps/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'daemonsets.apps' into namespace 'kube-system' from: /tmp/536302380/resources/daemonsets.apps/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'daemonsets.apps' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/daemonsets.apps/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'deployments.apps' into namespace 'test' from: /tmp/536302380/resources/deployments.apps/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Getting client for apps/v1, Kind=Deployment" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring Deployment: mysqlbackup" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'deployments.apps' into namespace 'heptio-ark' from: /tmp/536302380/resources/deployments.apps/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'deployments.apps' into namespace 'kube-system' from: /tmp/536302380/resources/deployments.apps/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'deployments.apps' into namespace 'rook-ceph' from: /tmp/536302380/resources/deployments.apps/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'deployments.apps' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/deployments.apps/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'endpoints' into namespace 'test' from: /tmp/536302380/resources/endpoints/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'endpoints' into namespace 'heptio-ark' from: /tmp/536302380/resources/endpoints/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'endpoints' into namespace 'kube-system' from: /tmp/536302380/resources/endpoints/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'endpoints' into namespace 'rook-ceph' from: /tmp/536302380/resources/endpoints/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'ingresses.extensions' into namespace 'test' from: /tmp/536302380/resources/ingresses.extensions/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'jobs.batch' into namespace 'rook-ceph' from: /tmp/536302380/resources/jobs.batch/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring cluster level resource 'podsecuritypolicies.extensions' from: /tmp/536302380/resources/podsecuritypolicies.extensions/cluster" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring cluster level resource 'podsecuritypolicies.policy' from: /tmp/536302380/resources/podsecuritypolicies.policy/cluster" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'podvolumebackups.ark.heptio.com' into namespace 'heptio-ark' from: /tmp/536302380/resources/podvolumebackups.ark.heptio.com/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'pools.ceph.rook.io' into namespace 'rook-ceph' from: /tmp/536302380/resources/pools.ceph.rook.io/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring cluster level resource 'priorityclasses.scheduling.k8s.io' from: /tmp/536302380/resources/priorityclasses.scheduling.k8s.io/cluster" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'prometheuses.monitoring.coreos.com' into namespace 'test' from: /tmp/536302380/resources/prometheuses.monitoring.coreos.com/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'prometheusrules.monitoring.coreos.com' into namespace 'test' from: /tmp/536302380/resources/prometheusrules.monitoring.coreos.com/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'replicasets.apps' into namespace 'test' from: /tmp/536302380/resources/replicasets.apps/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="default/mysqlbackup-6b7bbbcb84 has a controller owner - skipping" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'replicasets.apps' into namespace 'heptio-ark' from: /tmp/536302380/resources/replicasets.apps/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'replicasets.apps' into namespace 'kube-system' from: /tmp/536302380/resources/replicasets.apps/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'replicasets.apps' into namespace 'rook-ceph' from: /tmp/536302380/resources/replicasets.apps/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'replicasets.apps' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/replicasets.apps/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'resticrepositories.ark.heptio.com' into namespace 'heptio-ark' from: /tmp/536302380/resources/resticrepositories.ark.heptio.com/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'rolebindings.rbac.authorization.k8s.io' into namespace 'test' from: /tmp/536302380/resources/rolebindings.rbac.authorization.k8s.io/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'rolebindings.rbac.authorization.k8s.io' into namespace 'kube-public' from: /tmp/536302380/resources/rolebindings.rbac.authorization.k8s.io/namespaces/kube-public" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'rolebindings.rbac.authorization.k8s.io' into namespace 'kube-system' from: /tmp/536302380/resources/rolebindings.rbac.authorization.k8s.io/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'rolebindings.rbac.authorization.k8s.io' into namespace 'rook-ceph' from: /tmp/536302380/resources/rolebindings.rbac.authorization.k8s.io/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'rolebindings.rbac.authorization.k8s.io' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/rolebindings.rbac.authorization.k8s.io/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'roles.rbac.authorization.k8s.io' into namespace 'test' from: /tmp/536302380/resources/roles.rbac.authorization.k8s.io/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'roles.rbac.authorization.k8s.io' into namespace 'kube-public' from: /tmp/536302380/resources/roles.rbac.authorization.k8s.io/namespaces/kube-public" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'roles.rbac.authorization.k8s.io' into namespace 'kube-system' from: /tmp/536302380/resources/roles.rbac.authorization.k8s.io/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'roles.rbac.authorization.k8s.io' into namespace 'rook-ceph' from: /tmp/536302380/resources/roles.rbac.authorization.k8s.io/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'roles.rbac.authorization.k8s.io' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/roles.rbac.authorization.k8s.io/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'schedules.ark.heptio.com' into namespace 'heptio-ark' from: /tmp/536302380/resources/schedules.ark.heptio.com/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'servicemonitors.monitoring.coreos.com' into namespace 'test' from: /tmp/536302380/resources/servicemonitors.monitoring.coreos.com/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'services' into namespace 'test' from: /tmp/536302380/resources/services/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'services' into namespace 'heptio-ark' from: /tmp/536302380/resources/services/namespaces/heptio-ark" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'services' into namespace 'kube-system' from: /tmp/536302380/resources/services/namespaces/kube-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'services' into namespace 'rook-ceph' from: /tmp/536302380/resources/services/namespaces/rook-ceph" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'statefulsets.apps' into namespace 'test' from: /tmp/536302380/resources/statefulsets.apps/namespaces/default" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring cluster level resource 'storageclasses.storage.k8s.io' from: /tmp/536302380/resources/storageclasses.storage.k8s.io/cluster" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:32:15Z" level=info msg="Restoring resource 'volumes.rook.io' into namespace 'rook-ceph-system' from: /tmp/536302380/resources/volumes.rook.io/namespaces/rook-ceph-system" logSource="pkg/restore/restore.go:342"
time="2018-10-02T20:48:40Z" level=error msg="unable to successfully complete restic restores of pod's volumes" error="pod volume restore failed: error restoring volume: error running restic restore, cmd=restic restore --repo=s3:http://minio:9000/tarioark-restic/test --password-file=/tmp/ark-restic-credentials-test795185276 --cache-dir=/scratch/.cache/restic 3548b30b --target=., stdout=, stderr=invalid id \"3548b30b\": no ID found\n: exit status 1" logSource="pkg/restore/restore.go:856"
```

**Environment:**

- Ark version (use `ark version`):  
```
Version: v0.9.6
Git commit: v0.9.6
Git tree state: clean
```
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.3", GitCommit:"a4529464e4629c21224b3d52edfe0ea91b072862", GitTreeState:"clean", BuildDate:"2018-09-09T18:02:47Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.3", GitCommit:"a4529464e4629c21224b3d52edfe0ea91b072862", GitTreeState:"clean", BuildDate:"2018-09-09T17:53:03Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
```
- Kubernetes installer & version:
```
kubeadm version: &version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.3", GitCommit:"a4529464e4629c21224b3d52edfe0ea91b072862", GitTreeState:"clean", BuildDate:"2018-09-09T17:59:42Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
```
- Cloud provider or hardware configuration:
Using installation on debian stretch, with rook.io as storage provider, minio as a b2 gateway.

**Comment by skriss** [comment]
@tarioch unfortunately you've run into a bug w/restic restores not playing nicely with remapped namespaces, that I've fixed in a not-yet-merged PR (see https://github.com/heptio/ark/pull/888/commits/d0411f12eaa720cdd421f1c4f331bf6ffe7876ee).  The restore should work fine if you don't use the namespace remappings.

We could probably get this bug-fix out in our next v0.9 bug-fix release, which is scheduled for this Thursday.  We'll keep you updated here.

**Comment by skriss** [comment]
@tarioch look for a `v0.9.7` release tomorrow that contains this fix.
